GUI/QtQuick/qml/WalletProperties/WalletPropertiesPopup.qml

```diff
@@ -29,6 +29,11 @@ CustomPopup {
         visible: false
 
         wallet_properties_vm: root.wallet_properties_vm
+
+        onSig_success: (nameExport) => {
+            _stack_view.push(success)
+            success.details_text = qsTr("Wallet %1 has successfully been exported").arg(nameExport)
```

Comment:
Looks like destination path is missing here. I suggest "Wallet %1 has successfully been exported to %2", where %2 is the path.